src/api/.rubocop_todo.yml

```diff
@@ -1347,11 +1364,18 @@ Style/ConditionalAssignment:
     - 'spec/support/database_cleaner.rb'
     - 'test/functional/source_controller_test.rb'
 
-# Offense count: 841
+# Offense count: 843
 Style/Documentation:
   Enabled: false
 
-# Offense count: 1429
+# Offense count: 2
+# Cop supports --auto-correct.
```

Comment:
@saraycp there are still a few new additions to rubocop_todo that support autocorrection. Is this by purpose?

Reply:
I ran auto-correct before updating the `.rubocop_todo.yml`file. However, some of the cops, like this one, weren't corrected.
After a bit of research, I found the reason.

Some cops look like this:
![Screenshot_2020-08-17 Style RuboCop Docs](https://user-images.githubusercontent.com/2581944/90374587-0270da80-e074-11ea-907e-1d4fdbd0caf8.png)

If they contain "Yes (Unsafe)" instead of simply "Yes" for auto-correction, then the correction won't be applied. It is explained [here](https://docs.rubocop.org/rubocop/0.87/usage/auto_correct.html#safe-auto-correct):

> If a cop or its auto-correct is annotated as "not safe", it will be omitted when using --auto-correct.

In this case, I'd keep it in the TODO file and will tackle these errors manually in the future: Style/ExplicitBlockArgument, Style/SingleArgumentDig and Style/StringConcatenation.